community/kernel/src/main/java/org/neo4j/tooling/GlobalGraphOperations.java

```diff
@@ -155,12 +160,37 @@ protected Relationship fetchNextOrNull()
      */
     public Iterable<RelationshipType> getAllRelationshipTypes()
     {
+        return getAllRelationshipTypes( false );
+    }
+
+    public Iterable<RelationshipType> getAllRelationshipTypesInUse()
+    {
+        return getAllRelationshipTypes( true );
+    }
+
+    private Iterable<RelationshipType> getAllRelationshipTypes( boolean inUse )
+    {
         assertInUnterminatedTransaction();
-        return cast( relationshipTypes.getAllTokens() );
+        Iterable<RelationshipTypeToken> relationshipTypes = this.relationshipTypes.getAllTokens();
+        if ( inUse )
+        {
+            relationshipTypes = filter( new Predicate<Token>()
+            {
+                @Override
+                public boolean test( Token token )
+                {
+                    Statement statement = statementCtxSupplier.get();
```

Comment:
2 reasons:
- there is no API for collecting rel counts from the CountsStore 
- you have to load all token entries from the label/teltype store since you have to map ids to strings

I don't see the benefit of adding those API for this use case.